gridstatus/tests/test_miso.py

```diff
@@ -81,13 +81,75 @@ def test_get_load_historical_with_date_range(self):
 
     """get_load_forecast"""
 
+    load_forecast_cols = [
+        "Interval Start",
+        "Interval End",
+        "Publish Time",
+        "LRZ1 MTLF (MWh)",
```

Comment:
i think we can get rid of the `(MWh)`. we dont include that in any other dataset